### DISCUSSION TITLE
Minimal bounding box of circular_arc<Exact_circular_kernel_2>

### DISCUSSION THREAD

**Discussion by young03600** [CGAL/cgal#6814] [Q&A]
I used the arc access function to get the bounding box, but it seems that the given bounding box of the arc is not minimal
E.g. source point of arc: 343.2575,232.4850, target point: 388.2769,219.7406, center: 317.7468,265.8458
the given bounding box is (points from left to right, bottom to top)
```
{
    268.637,207.147,
    347.008,207.147,
    347.008,285.518,
    268.637,285.518
}
```
Is there any way to get the minimal bounding box of circular_arc?
Any suggestions will be appreciated, thanks

**Comment by afabri** [comment]
Can you please put a minimal code example on gist.github.com.
